Thumbprint/Targets/Thumbprint/Components/LabeledCheckbox.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+/**
+ A labeled checkbox.
+
+ While we don't need to store new data over its superclass, the implementation of ThumbprintPlayground requires it to be its own, non-generic class.
```

Comment:
Another option would be to just define this class inside of ThumbprintPlayground. E.g. if you otherwise would've preferred to have people instantiating `LabeledControl<Checkbox>` directly, or with some decorator method, we could still use that as the main public API, and then move this `LabeledCheckbox` class internal to the playground (and maybe rename to something like `PlaygroundLabeledCheckbox` for clarity). Similar to how `PlaygroundColorView` is just a wrapper around `Color` to conform to the needs of the playground. I have no preference either way, just an idea.

Reply:
Uhm could be a good idea but I'd rather wait until Thumbprint playground lives in the Thumbprint repo before trying.